modules/cf-core/src/protocol/install-virtual-app.ts

```diff
@@ -1108,11 +1105,11 @@ async function getUpdatedStateChannelAndVirtualAppObjectsForInitiating(
   const initiatorAddress = xkeyKthAddress(initiatorXpub, 0);
   const intermediaryAddress = xkeyKthAddress(intermediaryXpub, 0);
 
-  const multisigAddressWithIntermediary = Store.getMultisigAddressWithCounterpartyFromMap(
+  const multisigAddressWithIntermediary = await Store.getMultisigAddressWithCounterpartyFromMap(
```

Comment:
we should replace the static function of the store class with a utils like one

Reply:
I'm onboard w this